tests/scripts/analyze_outcomes.py

```diff
@@ -280,7 +280,260 @@ def do_analyze_driver_vs_reference(outcome_file, args):
                 # Following test depends on AES_C
                 'test_suite_version': [
                     'Check for MBEDTLS_AES_C when already present',
-                ]
+                ],
+                # Following tests depends on PCKS7
+                'test_suite_pkcs12': [
+                    'PBE Decrypt, (Invalid padding & PKCS7 padding enabled)',
+                    'PBE Decrypt, pad = 7 (OK)',
+                    'PBE Decrypt, pad = 8 (Invalid output size)',
+                    'PBE Decrypt, pad = 8 (OK)',
+                    'PBE Encrypt, pad = 7 (OK)',
+                    'PBE Encrypt, pad = 8 (Invalid output size)',
+                    'PBE Encrypt, pad = 8 (OK)',
+                ],
+                # Following tests depends on PCKS7
+                'test_suite_pkcs5': [
+                    'PBES2 Decrypt (Invalid output size)',
```

Comment:
Note: out of scope (should be a separate PR), but I've been thinking for some time about adding the ability to ignore tests that match a regex. Here having just two lines for `PBES2 Decrypt` and `PBES2 Encrypt` would be more readable (and manageable, if we add more tests in the future).

Reply:
I would totally agree with that. This morning, while adding these "skip lines" I realized that most of the lines in `analyze_outcomes.py` script are for test being ignored...

My initial idea of improvement was to just specify to ignore tests based on a list of dependencies, but this would mean that for each disparity the script has to go to the corresponding `*.data` file, look for the disparity test and check its dependencies. However the main drawback of this approach would be that we do not have a direct control over which tests are skipped in the coverage comparison and which not. I mean, it's pretty easy to just select a dependency that skips a lot of test both in the accelerated version and in the reference one without realizing how many tests are skipped.

Your proposal looks simpler to implement and perhaps give the same amount of control we have now. I will create a new task for this.

Reply:
> ignore tests based on a list of dependencies

I think it was a deliberate choice not to do that, because the goal was to catch undesired dependencies, typically dependencies that we forgot to update. It could be ok if there was one specific dependency `SKIPPED_IF_DRIVER_ONLY`.

I think a regex is easier though. We do have to be disciplined enough to make the regexes both specific enough (to not catch things inadvertently) and readable (so we can easily tell if something is on the ignore list).

Reply:
Just created #8365 about that, and when coming back to add a link, I found out I had missed the last two comments here.

Reply:
> I think it was a deliberate choice not to do that, because the goal was to catch undesired dependencies, typically dependencies that we forgot to update. It could be ok if there was one specific dependency `SKIPPED_IF_DRIVER_ONLY`.

Indeed it was a deliberate choice, for this reason.

Reply:
@mpg is it me or #8365 is pretty similar to #8361?

Reply:
Yeah, sorry, I failed to notice you had already created 8361 when I created 8365, otherwise I'd have added my thoughts to 8361 as comments.

Reply:
No problem at all, I was only asking to understand if it was worth to keep #8361 around or not. Since it's not, then I will close it as duplicate